index.html

```diff
@@ -371,7 +371,7 @@ <h2>More information</h2>
         <div class="row">
           <div class="col-md-3 col-md-offset-3 col-sm-4 col-sm-offset-2 col-xs-12">
             <p class="copyright">
-              &copy; 2018 <a href="https://simplabs.com"><i class="simplabs"></i></a>
+              &copy; 2019 <a href="https://simplabs.com"><i class="simplabs"></i></a>
```

Comment:
This was wrong before and should actually say _"2016-2019"_ now…

Reply:
👍